contrib/pkginstaller/scripts/postinstall

```diff
@@ -6,3 +6,8 @@ echo "/opt/podman/bin" > /etc/paths.d/podman-pkg
 
 # make sure to ignore errors, this is not a hard requirement to use podman
 /opt/podman/bin/podman-mac-helper install || :
+
+# Add pointer to podman docs in manpath
+if [ ! -d "/usr/local/etc/man.d/podman.man.conf" ]; then
```

Comment:
What is the point of checking if not directory? This is a file and there is no reason for the if AFAICT so this should be dropped.

Reply:
i should be checking for the dir and if not present, add it